kvmagent/kvmagent/plugins/host_plugin.py

```
@@ -4358,6 +4360,110 @@ def detach_volume__path(self, req):
+    def reconcile_vm_ha_enabled_metadata_live(self, req):
+        cmd = jsonobject.loads(req[http.REQUEST_BODY])
+        rsp = kvmagent.AgentResponse()
+
+        never_stop_vm_uuids = set(cmd.neverStopVmUuids or [])
+        running_vm_uuids = self._get_running_vm_uuids_on_host()
+        failed_updates = []
+        for vm_uuid in running_vm_uuids:
+            enable_ha = vm_uuid in never_stop_vm_uuids
+            try:
+                self._setup_vm_ha_enabled_metadata_live(vm_uuid, enable_ha)
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)